Tests-Legacy/L0/MavenV3/responseFeed.json

```diff
@@ -7,11 +7,15 @@
             "code": 0,
             "stdout": "Maven version 1.0.0"
         },
-        "/home/bin/maven/bin/mvn -f pom.xml -s /tmp/settings.xml package": {
+        "/home/bin/maven/bin/mvn -f pom.xml package": {
```

Comment:
The changes in this file shouldn't be required but the tests are failing (and I assume the product feature is working since we don't have known outstanding issues). We did make some changes around how the effective pom is generated: https://github.com/Microsoft/azure-pipelines-tasks/pull/6737/files. As per the test run, none of that code is being run. I am still investigating.

Reply:
Is it possible that bumping the vsts-task-lib version (#7972) introduced this?

Alternatively, what's the difference between say V2 and V3 of Maven?  I think it's the default values for the tasks, which look to be what the sorts of 'fixes' are that you're having to put in these files.

Reply:
Main difference between Maven V2 and V3 is the default value of setting mavenFeedAuthenticate. V2 has it as true and V3 is false. But the test is setting the value explicitly to "true".

Reply:
This turned out to be a difference in the logic in how the packaging URL is setup. I updated the mock output to match the packaging URL to the collection URL, so the feeds will actually be detected in the tests.